assets/style/about/about.scss

```
@@ -6,9 +6,536 @@
+          }
+        }
+        img {
+          @media (max-width: $tablet-l) {
+            display: block;
+            max-width: 85%;
+          }
+          @media (max-width: $phone-l) {
+            display: none;
+          }
+        }
+      }
+      .main__wr-company-descr {
+        margin-bottom: 161px;
+        max-width: 1270px;
+        height: 400px;
+        @include default-p($text-blue-color);
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        @media (max-width: 1650px) {
+          max-width: 80%;
+        }
+        @media (max-width: $pc-m) {
+          max-width: 70%;
+          overflow: scroll;
+        }
+        @media (max-width: $laptop-m) {
+          max-width: 60%;
+          margin-bottom: 90px;
+        }
+        @media (max-width: $tablet-s) {
+          font-size: 18px;
+          max-width: 50%;
+        }
+        @media (max-width: $phone-l) {
+          max-width: 100%;
+          text-align: center;
+        }
+      }
+    }
+
+    .main__company-principles {
+      min-height: 890px;
+      height: 100%;
+      @media (max-width: $pc-s) {
+        min-height: 1200px;
+      }
+      @media (max-width: $tablet-l) {
+        min-height: 1500px;
+      }
+      @media (max-width: $tablet-s) {
+        min-height: 1700px;
+      }
+      @media (max-width: $phone-l) {
+        min-height: 1650px;
+      }
+      @media (max-width: $phone-m) {
+        min-height: 1550px;
+      }
+      .main__principles-title {
+        @include default-h($text-dark-color);
+        margin-bottom: 85px;
+        text-align: center;
+        @media (max-width: $tablet-l) {
+          margin-bottom: 40px;
+        }
+        @media (max-width: $tablet-l) {
+          font-size: 38px;
+        }
+        @media (max-width: $phone-m) {
+          font-size: 32px;
+        }
+      }
+      .main__wr-principles-box {
+        display: grid;
+        grid-template-columns: 510px 510px 510px;
+        grid-template-rows: 275px 275px;
+        gap: 35px;
+        margin-bottom: 146px;
+        @media (max-width: 1650px) {
+          grid-template-columns: minmax(350px, 1fr) minmax(350px, 1fr) minmax(
+              350px,
+              1fr
+            );
+        }
+        @media (max-width: $pc-s) {
+          grid-template-columns: minmax(250px, 1fr) minmax(250px, 1fr);
+          grid-template-rows: minmax(290px, 1fr) minmax(290px, 1fr) minmax(
+              290px,
+              1fr
+            );
+          margin-bottom: 0px;
+        }
+        @media (max-width: $tablet-l) {
+          grid-template-columns: minmax(250px, 1fr);
+          grid-template-rows:
+            minmax(210px, 1fr) minmax(210px, 1fr) minmax(210px, 1fr)
+            minmax(210px, 1fr) minmax(210px, 1fr) minmax(210px, 1fr);
+          gap: 25px;
+        }
+
+        .main__principles-box {
+          background-color: #f9fafd;
+          padding-left: 41px;
```

Как по мне, если уж внутри карточки есть отступ с одной стороны, то такой же отступ должен быть со всех сторон этой карточки.
```suggestion
          padding: 41px;
```